ext/mysqli/mysqli.stub.php

```diff
@@ -0,0 +1,548 @@
+<?php
+
+class mysqli_driver
+{
+}
+
+class mysqli
+{
+    public function __construct(?string $host = null, ?string $user = null, ?string $password = null, ?string $database = null, ?int $port = null, ?string $socket = null);
+
+    public function autocommit(?bool $mode): bool;
```

Comment:
Please don't specify return types for methods (this could break BC because of inheritance). Instead use doc-block annotations. For functions, return types are fine.

Reply:
fixed. Does this look ok now?

Reply:
Yes, it's good now, but please remove the unnecessary PHPDoc-based parameter type hints from everywhere.

Reply:
Just to be clear, just remove all docblocks? Because they are only present when the signature differs, eg: when the return type is omitted (always in the classes)

Reply:
The parameter types that are duplicated (those ones which are also present as native type declaration) should be removed from the phpdoc. Have a look at https://github.com/php/php-src/pull/4846/files for example.